### DISCUSSION TITLE
Hope that the system fields can be sorted and hidden

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#8921] [Archive]
For example, the field in directus_users.

**Comment by cdwmhcc** [comment]
![image](https://user-images.githubusercontent.com/8030134/137777122-a179c5c6-c21c-49e8-93e3-bfc53e4e7555.png)

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/7399
